src/app/services/theme/theme.service.ts

```diff
@@ -23,7 +23,13 @@ const themeVariants = [
   "-darker",
   "-darkest",
 ] as const;
-export type ThemeVariant = typeof themeVariants[number];
+export type ThemeVariant = (typeof themeVariants)[number];
+
+const bootstrapVariables: [string, number][] = [
+  ["--bs-link-color", 0],
+  ["--bs-link-hover-color", -0.1],
```

Comment:
i'd define the values with constants or a hash. Darkest, darker, normal, lighter, lightest - as we do in our css. Link back to those other definitions